.travis.yml

```diff
@@ -2,5 +2,4 @@ language: clojure
 lein: lein2
 script: lein2 test-all
 jdk:
-  - openjdk7
-  - openjdk6
+  - oraclejdk8
```

Comment:
Why did you change this? I can see why you removed JDK6, but I'm pretty sure ClojureScript works with JDK7?